force-app/main/default/lwc/propertyLocation/propertyLocation.html

```diff
@@ -0,0 +1,40 @@
+<template>
+    <lightning-card
+        title="Distance to the Property"
+        icon-name="standard:location"
+    >
+        <template if:false={error}>
+            <template if:true={distance}>
+                <div class="slds-var-m-around_medium location">
+                    <div class="slds-text-heading_small">
+                        Current latitude:
+                        <b class="latitude">{location.latitude}</b>
+                    </div>
+                    <div class="slds-text-heading_small">
+                        Current longitude:
+                        <b class="longitude">{location.longitude}</b>
+                    </div>
+                    <!-- prettier-ignore -->
```

Comment:
Can you try to get rid of the `prettier-ignore` tag?
Perhaps with two `&nbsp;` like this:
```html
<div class="slds-text-heading_small">
    Distance to property:&nbsp;<b>
        <lightning-formatted-number
            value={distance}
            maximum-fraction-digits="2"
        ></lightning-formatted-number>&nbsp;miles
    </b>
</div>
```

Feel free to ignore if this doesn't work.